lib/class/api.class.php

```diff
@@ -402,7 +530,16 @@ public static function albums($input)
     public static function album($input)
     {
         $uid = scrub_in($input['filter']);
-        echo XML_Data::albums(array($uid));
+
+        //Whatever format the user wants
+        $outputFormat = $input['format'];
+
+        if ($outputFormat == 'json') {
+            echo JSON_Data::albums(array($uid));
```

Comment:
Array declaration should be moved outside of the test and `*_Data::albums` should take a variable argument.